.github/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 🔥 Firebase Support
+    url: https://firebase.google.com/support/
+    about: If you have an urgent issue with your app, please contact support.
```

Comment:
@ashwinraghav I think you meant to create this file under `.github/ISSUE_TEMPLATE/`?
(As shown in the [Github Docs](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser))

Reply:
Yep. Realised that late Friday. Thanks 👍